src/gui/editors/PianoRoll.cpp

```diff
@@ -248,6 +249,15 @@ PianoRoll::PianoRoll() :
 	connect( markChordAction, &QAction::triggered, [this](){ markSemiTone(stmaMarkCurrentChord); });
 	connect( unmarkAllAction, &QAction::triggered, [this](){ markSemiTone(stmaUnmarkAll); });
 	connect( copyAllNotesAction, &QAction::triggered, [this](){ markSemiTone(stmaCopyAllNotesOnKey); });
+	connect( ConfigManager::inst(), &ConfigManager::valueChanged,
+		[this](QString const& cls, QString const& attribute, QString const& value)
+		{
+			if (cls != "midi" && attribute != "autoquantize")
+			{
+				return;
+			}
+			this->m_doAutoQuantization = (value.toInt() != 0);
```

Comment:
Same suggestion as before, again, without being sure whether it has been decided to avoid implicit conversions. 
```suggestion
			this->m_doAutoQuantization = value.toInt();
```

Reply:
Preferring explicit test for better readability.